File: 06 - Java/09-07-00-EjercicioPropuesto-ValorAbsoluto-ULP.java
import java.util.Scanner;

public class ProgramaPrincipal {

    public static void main(String[] args){
        System.out.print("Proporciona un valor numerico: ");
        var numero = Integer.parseInt(new Scanner(System.in).nextLine());
        var numeroAbs = Math.abs(numero);
        System.out.println("Valor absoluto de " + numero + " es: " + numeroAbs);
    }
}

File: 03 - C++/09-07-00-EjercicioPropuesto-ValorAbsoluto-ULP.cpp
#include <iostream>


using namespace std;

int main(){  
	int numero, numeroAbs;
	printf("Proporciona un valor numerico: ");
	cin >> numero;
	numeroAbs = abs(numero);
	cout << "Valor absoluto de " << numero << " es: " << numeroAbs;
	
	return 0;
}
	
